uvloop/sslproto.pxd

```diff
@@ -0,0 +1,113 @@
+cdef enum ProtocolState:
```

Comment:
I'd call it `SSLProtocolState`. And maybe we don't need the leading `_`?

Reply:
Right, they were globals and I forgot to rename them.